.vscode/settings.json

```diff
@@ -1,8 +1,8 @@
 {
   "files.eol": "\n",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
   },
   "editor.rulers": [ 140 ],
   "eslint.enable": true
-}
\ No newline at end of file
+}
```

Comment:
These changes are automatically being made by the latest version of VS Code.

Happy to take these out, but thought might as well merge them, as I imagine you'll face the same. See https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto

Reply:
Sounds ok to me – I don't use VS Code, these settings are actually part of the original homebridge template.